sdk/download.go

```diff
@@ -318,3 +319,18 @@ func UpdateSignedFile(file, url string, client *http.Client) error {
 	plog.Infof("Verified file: %s", file)
 	return nil
 }
+
+func DetectSDKVersion(url, branch string) (string, error) {
+	tmp, err := ioutil.TempDir("", "")
+	if err != nil {
+		return "", err
+	}
+	defer os.RemoveAll(tmp)
+
+	clone := exec.Command("git", "clone", "--depth=1", "--single-branch", "-b", branch, url, tmp)
```

Comment:
ok it looks like passing -q to git clone will avoid the extra noise:

```
 cork create --replace
2015-12-28T23:21:09Z cork: Detecting SDK version
2015-12-28T23:21:10Z cork: Downloading SDK version 891.0.0
```

and here it is when something is wrong

```
 cork create --replace --manifest-url=foo
2015-12-28T23:20:08Z cork: Detecting SDK version
fatal: repository 'foo' does not exist
2015-12-28T23:20:08Z cork: DetectSDKVersion failed: exit status 128
```